config/mainnet.go

```diff
@@ -160,14 +160,13 @@ func MainnetConfig() Config {
 			BeaconSyncWeightUnits:    800,
 		},
 		POET: activation.PoetConfig{
-			PhaseShift:                     240 * time.Hour,
-			CycleGap:                       12 * time.Hour,
-			GracePeriod:                    1 * time.Hour,
-			PositioningATXSelectionTimeout: 50 * time.Minute,
```

Comment:
This timeout seems to have disappeared

Also it doesn't seem to have been set for other presets like `fastnet` and `testnet`. Can you add it and make it a bit smaller than the `GracePeriod` for those presets? 🙂